www/~/%username/emails/index.spt

```diff
@@ -1,12 +1,18 @@
-from gratipay.utils import get_participant
+from gratipay.utils import get_participant, icons
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=True)
 banner = '~' + participant.username
-title = _("Emails")
+title = _("Email")
 
 emails = participant.get_emails()
+groups = [[], [], []]  # primary, verified, unverified
+for email in emails:
+    i = 0 if email.address == participant.email_address else 1 if email.verified else 2
+    groups[i].append(email)
+emails = sum(groups, [])
```

Comment:
😃 Is this common usage? 

Perils of not having `flatten` in the std lib? :P

Reply:
Yes, that's how to spell `flatten` in Python. 🐭